traffic_ops/client/cachegroup.go

```diff
@@ -189,3 +189,24 @@ func (to *Session) DeleteCacheGroupByID(id int) (tc.Alerts, ReqInf, error) {
 	}
 	return alerts, reqInf, nil
 }
+
+func (to *Session) GetCacheGroupsByQueryParams(qparams url.Values) ([]tc.CacheGroupNullable, ReqInf, error) {
```

Comment:
Nit: Go doc?

Reply:
added in 59d6fb1005576218c08246171707843ba05e3e6f